README.adoc

```diff
@@ -48,24 +49,21 @@ The HiveMQ MQTT Message Log Extension is a very useful HiveMQ Extension for debu
 |HiveMQ acknowledges the reception of a Qos2 publish with a pubrec message | pubrec-send | 4.3 EE or 2020.1 CE
 |HiveMQ completes a sent Qos2 publish with a pubrel message | pubrel-send | 4.3 EE or 2020.1 CE
 |HiveMQ completes a received Qos2 publish with a pubcomp message | pubcomp-send | 4.3 EE or 2020.1 CE
-
 |===
 
-=== Installation
+== Installation
 
-. Download https://www.hivemq.com/releases/extensions/hivemq-mqtt-message-log-extension-1.1.0.zip[HiveMQ Mqtt Message Log Extension]
-.. Move the file: "hivemq-mqtt-message-log-extension-1.1.0.zip" to the directory: "HIVEMQ_HOME/extensions"
-. Or clone this repository into a Java 11 maven project
-.. run `mvn package` goal from Maven to build the extension.
-.. Move the file: "hivemq-mqtt-message-log-extension-1.1.0-distribution.zip" to the directory: "HIVEMQ_HOME/extensions"
-. Unzip the file.
-. Start HiveMQ.
+* Download the extension from the {hivemq-extension-downloads}[HiveMQ Marketplace^].
```

Comment:
why do we not link directly to https://www.hivemq.com/extension/mqtt-message-log-extension/?